### ISSUE TITLE
Can we have the "potato mode" again? 

### ISSUE THREAD

**Issue by anonymousDog12** [HasangerGames/suroi#377]
**Is your feature request related to a problem? Please describe.**

Many players and I miss the potato mode from surviv.io. I personally also miss cobalt mode. 

**Describe the solution you'd like**

Have the potato mode.

**Comment by SurvivWasTheBestGame** [comment]
Yes, we should add them, also vote for which modes we should add in the discussions tab.
